src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -58,6 +58,53 @@ public void GetProcessesByName_RemoteMachineNameUnix_ThrowsPlatformNotSupportedE
             Assert.Throws<PlatformNotSupportedException>(() => Process.GetProcessesByName(currentProcess.ProcessName, machineName));
         }
 
+        [Fact]
```

Comment:
Would this work on windows too? If so we might as well run it there

Reply:
could work...will do

Reply:
C:\Program Files (x86)\Windows Resource Kits\Tools\sleep.exe on my machine....I think it's installed also on some CI machine...BTW in that case will skip.

Reply:
Ah yes it would probably have to be notepad or something in that case - probably not worth the bother.